hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -881,29 +881,21 @@ The table also lists which error codes are supported by which agent.
-0x4        | OtpReadUncorrErr | no          |  x  |  -  |   x   |  x    | Uncorrectable ECC error occurred during a read in the OTP macro.
-0x5        | OtpReadErr       | no          |  x  |  -  |   x   |  x    | An unrecoverable error occurred during a read operation in the OTP macro.
-0x6        | OtpWriteBlankErr | yes         |  x  |  x  |   -   |  -    | OTP macro blank check failed. The location to be programmed is not all-zero.
-0x7        | OtpWriteErr      | no          |  x  |  x  |   -   |  -    | An unrecoverable error occurred during a program operation in the OTP macro.
-0x8        | CmdInvErr        | yes         |  x  |  -  |   -   |  -    | An invalid command has been issued.
-0x9        | AccessErr        | yes         |  x  |  -  |   x   |  -    | An access error has occurred (e.g. write to write-locked region, or read to a read-locked region).
-0xA        | ParityErr        | no          |  -  |  -  |   x   |  x    | A parity error has been detected.
-0xB        | IntegErr         | no          |  -  |  -  |   -   |  x    | An integrity error has been detected.
-0xC        | CnstyErr         | no          |  -  |  -  |   -   |  x    | A consistency error has been detected.
-0xD        | FsmErr           | no          |  x  |  x  |   x   |  x    | The FSM has been glitched into a parasitic state.
-0xE        | EscErr           | no          |  x  |  x  |   x   |  x    | Escalation has been triggered and the FSM has been moved into a terminal error state.
-0xF        | -                | -           |  -  |  -  |   -   |  -    | Reserved.
+Note that error codes that originate in the physical OTP macro are prefixed with `Macro*`.
+
+Error Code | Enum Name            | Recoverable | DAI | LCI | Unbuf | Buf   | Description
+-----------|----------------------|-------------|-----|-----|-------|-------|-------------
+0x0        | NoError              | -           |  x  |  x  |   x   |  x    | No error has occurred.
+0x1        | MacroError           | no          |  x  |  x  |   x   |  x    | Returned if the OTP macro command was invalid or did not complete successfully due to a macro malfunction.
+0x2        | MacroEccCorrError    | yes         |  x  |  -  |   x   |  x    | A correctable ECC error has occurred during a read operation in the OTP macro.
+0x3        | MacroEccUncorrError  | no          |  x  |  -  |   x   |  x    | An uncorrectable ECC error has occurred during a read operation in the OTP macro.
+0x4        | MacroWriteBlankError | yes         |  x  |  x  |   x   |  x    | This error is returned if a write operation attempted to overwrite an already programmed location.
+0x5        | AccessError          | yes         |  x  |  -  |   x   |  -    | An access error has occurred (e.g. write to write-locked region, or read to a read-locked region).
+0x6        | CheckFailError       | no          |  -  |  -  |   x   |  x    | An unrecoverable parity, integrity or consistency error has been detected.
```

Comment:
@cdgori @tjaychen are we ok with consolidating all these errors (Parity, integrity, consistency) into one error code?

I did that mainly in order to get the number of error codes down to 8, and since all these three errors basically lead to the same outcome (alert, and then escalation).

Reply:
If they all follow the same resolution path then I think it is ok.  Do you expect SW to take any different corrective action between those 3 things or is it exactly the same?

Reply:
It doesn't matter from a SW perspective. I initially split out for informational reasons only.